roles/firewall/tests/test.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  remote_user: root
+  roles:
+    - Firewall
```

Comment:
tpt eixe ena kokkino eikonidio sto telos pou elege "no new line at the end of file"